karma.conf.js

```diff
@@ -0,0 +1,344 @@
+// Karma configuration
+// Generated on Tue Oct 03 2017 13:01:40 GMT-0700 (PDT)
+
+var local = false
```

Comment:
Possible to use `let` here?

Reply:
Got it.  Should it be okay since we use node@6?